Cargo.toml

```diff
@@ -23,6 +23,10 @@ semver = "0.5.0"
 separator = "0.3.1"
 time = "0.1.35"
 
+[dependencies.clippy]
+optional = true
+version = "0.0.118"
```

Comment:
hm, I'm a little opposed since I'd rather have predictably passing travis builds, instead of either a. waiting for fixes or b. merging PRs with stylistic issues. If new versions bubble up new warnings, it will also impede people who want to make PRs on unrelated things. 

It requires more upkeep to update it constantly, though, so I would be cool with it. Do you know if new versions can bring up warnings on previously-passing code?